src/Contao/View/Contao2BackendView/Subscriber/GetGroupHeaderSubscriber.php

```diff
@@ -245,6 +249,25 @@ private function formatByDayGrouping($value)
         return $event->getResult();
     }
 
+    /**
+     * Render a grouping header for week.
+     *
+     * @param int $value The value.
+     *
+     * @return string
+     */
+    private function formatByWeekGrouping($value)
+    {
+        $value = $this->getTimestamp($value);
+        if ('' === $value) {
+            return '-';
+        }
+        $event = new ParseDateEvent($value, 'W. Y');
```

Comment:
Adjust the output of the new grouping. The monthly grouping already exists. This output should not differ.

Reply:
The `W.` creates the numer of the number of the week in current year.

Reply:
I meant please remove the dot.

Reply:
hmm... 
with the month, the issue is not the number, but the name... I'm not sure if it's better with or without a dot...

Reply:
I think without the dot we need somtehing else (e.g. KW as prefix).

Reply:
We also do not have a divider or anything like that when grouping for month. If then one should also adjust. So that it is equally formatted.

@discordier What do you think?

Reply:
I think the discussion could be endless. I never define the week in german without the dot. But lets find a solution. What about a simple configuration? We can add a key to the translations with the default format and everybody is abled to change it.

Reply:
@cliffparnitzky Sorry, I misread the format of the grouping for the month.

Reply:
we should add a default date format and override this with date format from language file

Reply:
I am also all for providing/using language keys for the formats.

Reply:
@cliffparnitzky can you change your PR?

Reply:
MM call: ISO8601 reicht!